static/menu.js

```diff
@@ -0,0 +1,6 @@
+let menu = document.getElementById("collapsible-menu");
```

Comment:
This should probably be in a `DOMContentLoaded` event handler, like the other scripts. (And have `'use strict';`.)